#! /bin/sh -e
## ubuntu-disable-browsing.dpatch by Martin Pitt <martin.pitt@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Disable Browsing by default on Ubuntu (not sanctioned yet by
## DP: https://wiki.ubuntu.com/DefaultNetworkServices). This is only applied
## DP: when building on Ubuntu.

dpkg-vendor --is ubuntu || { echo -n '(ignored)' >&2; exit 0; }
. /usr/share/dpatch/dpatch-run

@DPATCH@
diff -urNad cups-1.4.0~svn8773~/conf/cupsd.conf.in cups-1.4.0~svn8773/conf/cupsd.conf.in
--- cups-1.4.0~svn8773~/conf/cupsd.conf.in	2009-08-23 12:15:52.000000000 +0200
+++ cups-1.4.0~svn8773/conf/cupsd.conf.in	2009-08-23 19:08:47.000000000 +0200
@@ -19,10 +19,11 @@
 @CUPS_LISTEN_DOMAINSOCKET@
 
 # Show shared printers on the local network.
-Browsing On
+Browsing Off
 BrowseOrder allow,deny
 BrowseAllow all
 BrowseLocalProtocols @CUPS_BROWSE_LOCAL_PROTOCOLS@
+BrowseAddress @LOCAL
 
 # Default authentication type, when authentication is required...
 DefaultAuthType Basic
